distributed/scheduler.py

```diff
@@ -158,6 +169,7 @@ def nogil(func):
 
 
 LOG_PDB = dask.config.get("distributed.admin.pdb-on-err")
+ZEROCONF = dask.config.get("distributed.scheduler.zeroconf")
```

Comment:
I typically recommend to not define config options as globals. Defining them in globals sets them in stone after first module import. In particular any `dask.config.set(...)` ctx managers will be ignored which is often unexpected for users. I would actually expect your test to still pass even if you set the config to false